src/Sylius/Bundle/ApiBundle/Resources/translations/validators.en.yaml

```
@@ -7,6 +7,8 @@ sylius:
         without_country: 'The address without country cannot exist'
     country:
         not_exist: 'The country %countryCode% does not exist.'
+    locale:
+        not_exist: 'The locale %localeCode% does not exist.'
```

How about we provide an example.. like:
```suggestion
        not_exist: 'The locale %localeCode% does not exist. Try en_US'
```

---

We could put the parameter into validation error and plug default or all available locales into it

```suggestion
        not_exist: 'The locale %localeCode% does not exist. Try one of %possibleLocaleCodes%'
```